README.md

```diff
@@ -54,9 +54,7 @@ It makes use of [cairo-rs](https://github.com/lambdaclass/cairo-rs), the Rust im
 Run the following make targets to have a working environment (if in Mac or if you encounter an error, see the subsection below):
 ```bash
 $ make deps
-$ source starknet-venv/bin/activate
 $ make compile-cairo
```

Comment:
The _build_ target has a dependency on _compile-cairo_, so we can also remove this line.

Reply:
Adressed @ d8a10ce